config/hydroshare-config.yaml

```diff
@@ -1,18 +1,18 @@
-### Local Configuration Variables ###
-HS_PATH: /home/hydro/hydroshare
-HOST_SSL_DIR: /home/hydro/hs-certs
-HS_DATABASE: pg.development.sql
-HS_LOG_FILES: /home/hydro/hydroshare/log
+ ### Local Configuration Variables ###
+ HS_PATH: /home/hydro/hydroshare
+ HOST_SSL_DIR: /home/hydro/hs-certs
+ HS_DATABASE: pg.www.sql
+ HS_LOG_FILES: /home/hydro/hydroshare/log
 
-### Deployment Options ###
-USE_NGINX: false
-USE_SSL: false
+ ### Deployment Options ###
+ USE_NGINX: false
+ USE_SSL: false
 
-### nginx Configuration Variables ###
-FQDN_OR_IP: localhost
-NGINX_DIR: /home/hydro/hydroshare/nginx
+ ### nginx Configuration Variables ###
+ FQDN_OR_IP: localhost
+ NGINX_DIR: /home/hydro/hydroshare/nginx
 
-### SSL Configuration Variables ###
-SSL_CERT_DIR: /home/hydro/hydroshare/nginx/cert-files
-SSL_CERT_FILE: hydrodev-vb.example.org.cert
-SSL_KEY_FILE: hydrodev-vb.example.org.key
+ ### SSL Configuration Variables ###
+ SSL_CERT_DIR: /home/hydro/hydroshare/nginx/cert-files
+ SSL_CERT_FILE: hydrodev-vb.example.org.cert
+ SSL_KEY_FILE: hydrodev-vb.example.org.key
```

Comment:
Not sure why all these got indented one space but unnecessary changes should not be included in the branch.